### DISCUSSION TITLE
Module not found: Error: Can't resolve 'fs' in '/vercel/path0/node_modules/jpeg-exif/lib'

### DISCUSSION THREAD

**Discussion by SanjayPaul4u** [vercel/community#5546] [Help]
### Summary

This error comes on build time. and jpeg-exif module does not exits in package.json .
How can i resolve this issue 

### Steps to Reproduce

Failed to compile.
--
23:14:32.546 |  
23:14:32.546 | Module not found: Error: Can't resolve 'fs' in '/vercel/path0/node_modules/jpeg-exif/lib'
23:14:32.546 |  
23:14:32.547 |  
23:14:32.664 | error Command failed with exit code 1.
23:14:32.664 | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
23:14:32.689 | Error: Command "yarn run build" exited with 1

**Comment by makarenos** [comment]
same problem, tried everything, it works on local, but it won't upload to git

**Reply by makarenos** [reply]
writing exceptions in package.json doesn't help either

**Comment by makarenos** [comment]
Bellow solution worked for me.

npm i @craco/craco or yarn add @craco/craco
Create craco.config.js in the root of your project.
Bellow code in craco.config.js
module.exports = {
webpack: {
configure: (webpackConfig, { env, paths }) => {
webpackConfig.resolve.fallback = {
fs: false,
os: false,
path: false,
};
return webpackConfig;
},
},
};
modify your package.json
"build": "craco build",
"start": "craco start",
"test": "craco test",
yarn install or npm update

**Comment by makarenos** [comment]
https://github.com/diegomura/react-pdf/issues/2541
